apps/document-storage/.env.development

```diff
@@ -0,0 +1,23 @@
+#!bin/sh
+# - - - - - - - - Global - - - - - - - -
+
+# Node environments
+# Example : production, development
+NODE_ENV=development
+
+# - - - - - - - - Server - - - - - - - -
+
+# HTTP port
+# Default : 3001
+PORT=3001
+# Default : 3443
+HTTPS_PORT=3443
+
+# - - - - - - - - - Swagger - - - - - - - 
+
+SWAGGER_JSON_DIR=./src/server/swagger-output.json
+
+# - - - - - - - Blob Storage - - - - - -
+
+AZURE_BLOB_STORE_CONNECTION_STRING=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1;QueueEndpoint=http://localhost:10001/devstoreaccount1;
```

Comment:
This key should not be here.

Reply:
Any secret should be set in the gitignored file. Here is just for reference.

Reply:
This secret key actually not secret at all. This is the same secret key that everyone gets when using the local version of the azure blob storage. Here is a public page with all this info: https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string

In the dev/test envs, we will just need to override this connection string.